api/doris/v1/types.go

```diff
@@ -207,6 +207,10 @@ type BaseSpec struct {
 	//cnEnvVars is a slice of environment variables that are added to the pods, the default is empty.
 	EnvVars []corev1.EnvVar `json:"envVars,omitempty"`
 
+	//EnvFrom is a list of sources to populate environment variables in the container, the default is empty.
+	//+optional
+	EnvFrom []corev1.EnvFromSource `json:"envFrom,omitempty"`
```

Comment:
The 'EnvFrom' defined in DCR is not passed into the actual 'PodTemplateSpec' construction